BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -53,14 +54,18 @@ class SideMenuFilterViewController: UITableViewController {
     }
     
     @IBAction func sliderDidSlide(_ sender: UISlider) {
-
+        
+        UserPreferences.instance.distanceFilter.subtract(distanceFilter)
```

Comment:
+ I looked into this, it is a bit confusing 
+ Although it is necessary 
+ When removed the distance filter and the StoresTableViewController does not work properly